static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner-parameters.js

```diff
@@ -56,22 +53,78 @@ const supporterCost = (location: string): string => {
             : euro + amount;
     }
 
-    return `${extendedCurrencySymbol[countryGroup]}${initialContribution}`;
+    return `${extendedCurrencySymbol[countryGroup]}${amount}`;
 };
 
 const supporterEngagementCtaCopyControl = (location: string): string =>
     `<span class="engagement-banner__highlight"> Support The Guardian from as little as ${supporterCost(
-        location
+        location,
+        initialContribution
     )}.</span>`;
 
-export const engagementBannerParams = (
-    location: string
-): EngagementBannerParams =>
-    Object.assign({}, baseParams, {
+export const defaultEngagementBannerParams = (): EngagementBannerParams => {
+    const location = getGeoLocation();
+    return {
+        campaignCode: 'gdnwb_copts_memco_banner',
         buttonCaption: 'Support The Guardian',
         linkUrl: supportContributeURL,
-        products: ['CONTRIBUTION', 'RECURRING_CONTRIBUTION'],
-        messageText: engagementBannerCopy,
+        messageText: engagementBannerControl,
         ctaText: supporterEngagementCtaCopyControl(location),
         pageviewId: config.get('ophan.pageViewId', 'not_found'),
-    });
+        products: ['CONTRIBUTION', 'RECURRING_CONTRIBUTION'],
```

Comment:
why are we specifying products here, but not when building up the variants from google docs? I don't think we ever use products (@jranks123 ?); we could just omit them entirely.

Reply:
The products field is a wasteland of inaccurate data at this point, so we may as well send through an empty array, or remove it from the params altogether. For the scope of this PR it's probably fine to keep it as it is and deal with it separately.